src/UIManager.ts

```diff
@@ -481,7 +481,7 @@ export class UIManager {
         $(clone)
           .empty()
           .removeAttr("class")
-          .removeAttr("title")
+          .attr("title", "Feedly filtering and sorting")
```

Comment:
This does not work because title is a property and not an attribute (see https://stackoverflow.com/a/987974).
Use `.prop` instead.

Reply:
Interesting ... I didn't notice that as it seems to work here, anyway, I'll take a look and revise it properly ;)

Reply:
Maybe it does not work just on Firefox (even with `.prop`)

Reply:
From the examples in this page: https://api.jquery.com/attr/, `attr()` seems to work?

Reply:
That's more interesting as I tested it with my Firefox v66 😆

Reply:
Yeah, I think that in this case, the `title` attribute is set here before being added to the DOM, that's why it works :)